Showcase-iOS/AppDelegate.swift

```diff
@@ -8,14 +8,16 @@
 
 import UIKit
 import Firebase
+import IQKeyboardManager
+
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         FirebaseApp.configure()
+        IQKeyboardManager.shared().isEnabled = true
```

Comment:
This is cool and easy to use, but I would hide this behind an interface though just so that if we want to remove this lib or something we will be able to just swap out implementations, but its cool for now. Just a thought.

Reply:
Cool.

Reply:
@VaalSquared Let's remember to add to the tech debt.